tests/unit/framework/mocks.py

```diff
@@ -52,3 +53,14 @@ def rows_written_for(self, full_name: str, mode: str) -> list[DataclassInstance]
                 continue
             rows += stub_rows
         return rows
+
+
+class MockRuntimeBackend(MockBackend):
```

Comment:
no need to do this. remove.

see https://github.com/databrickslabs/ucx/blob/main/tests/unit/framework/test_crawlers.py#L167-L246

Reply:
Against, you don't need it

Reply:
Removed

Reply:
This is not removed.

Reply:
Swear to god it was removed